packages/palette-docs/content/docs/elements/images/Image.mdx

```diff
@@ -19,3 +19,15 @@ Used to fill an area provided regardless of proportions.
     <ResponsiveImage src="https://picsum.photos/400/250/?random" />
   </BorderBox>
 </Playground>
+
+### Lazy Image
+
+Useful for images that render below the fold
+
+<Playground>
+  <LazyImage
+    width="300px"
+    height="200px"
+    src="http://deelay.me/3000/https://picsum.photos/300/200/?random"
+  />
+</Playground>
```

Comment:
@zephraph - can you remove the lazy component in favor of the regular `<Image>` component with a `lazyLoad` prop? That or add and then move `LazyImage` down down with the disclaimer that its a lower-level primitive and that the regular `<Image>` component should be used. We should keep the number of components in circulation to a minimum if possible.